html/changelogs/sleepygemmy-messy_alt.yml

```
@@ -0,0 +1,6 @@
+author: SleepyGemmy
+
+delete-after: True
+
+changes:
+  - imageadd: "Added a variant of the \"Messy\" hairstyle called \"Messy, Alt\"."
```

```suggestion
  - rscadd: "Added a variant of the \"Messy\" hairstyle called \"Messy, Alt\"."
```
It's not just an image